common/src/main/java/io/netty/util/concurrent/GlobalEventExecutor.java

```diff
@@ -65,6 +65,8 @@ public void run() {
 
     private GlobalEventExecutor() {
```

Comment:
why does shutdown throw UnsupportedOperationException? can we tweak the lifecycle to invoke clear on the thread local in ThreadExecutorMap?

Reply:
@johnou because you never shutdown by yourself. The used Thread will die after some time by itself which also means it will be removed from the map

Reply:
@normanmaurer just wanted to raise a discussion, I was thinking of the use case where people redeploy webapps and it complains about thread locals / leaking memory - not a problem for us :wink: